libs/remix-simulator/src/methods/accounts.ts

```diff
@@ -85,14 +85,20 @@ export class Web3Accounts {
       eth_sign: this.eth_sign.bind(this),
       eth_chainId: this.eth_chainId.bind(this),
       eth_signTypedData: this.eth_signTypedData_v4.bind(this), // default call is using V4
-      eth_signTypedData_v4: this.eth_signTypedData_v4.bind(this)
+      eth_signTypedData_v4: this.eth_signTypedData_v4.bind(this),
+      eth_getPKey: this.eth_getPKey.bind(this),
     }
   }
 
   eth_requestAccounts (_payload, cb) {
     return cb(null, Object.keys(this.accounts))
   }
 
+  eth_getPKey (payload, cb) {
```

Comment:
where is this required?

Reply:
look for `await ethersProvider.send('eth_getPKey'`